syntax/odin.vim

```diff
@@ -60,7 +60,8 @@ syntax keyword odinDefer defer
 syntax region odinChar start=/\v'/ skip=/\v\\./ end=/\v'/
 syntax region odinString start=/\v"/ skip=/\v\\./ end=/\v"/
 
-syntax match odinFunction "\v<\w*>(\s*::\s*proc)@="
+syntax match odinFunctionDecl "\v<\w*>(\s*::\s*proc)@="
+syntax match odinFunctionCall    "\w\(\w\)*("he=e-1,me=e-1
```

Comment:
Instead of this `he=e-1,me=e-1` weirdness, lets use lookahead instead:
```diff
-syntax match odinFunctionCall    "\w\(\w\)*("he=e-1,me=e-1
+syntax match odinFunctionCall "\v\w+\s*(\()@="
```
`@=` means _Lookahead for the group immediately before me_.  [`(\()`]